### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by nbarbettini**

### CODE REVIEW

**Comment by nbarbettini** [comment]
@omgitstom anything I missed in the readme?

File: README.MD [hunk 1]
```diff
@@ -14,19 +12,235 @@ library gives your ASP.NET web application access to all of Stormpath's features
...
+- Easy installation using NuGet
+- Fully asynchronous design using the [TPL](https://msdn.microsoft.com/en-us/library/dd460717(v=vs.110).aspx)
+
 If you have feedback about this library, please get in touch and share your
 thoughts! support@stormpath.com
 
 ## Current status
 
-This library is currently under initial development. If you would like to be notified when we publish our first release candidate, watch this repository or send an email to nate@stormpath.com.
+This library is currently under active development. If you would like to be notified when we publish our first release candidate, watch this repository or send an email to nate@stormpath.com.
```
Comment (omgitstom) [inline README.MD]: We should funnel them to support@stormpath.com.  This is easier for the team to search, and aggregate information.  This isn't possible from your email.

Comment (nbarbettini) [inline README.MD]: Will revise. :+1:

**Comment by omgitstom** [comment]
Just had one inline comment.  The other thing, is that you hop from first-person plural to second person singular.  This is a problem with our documentation in general, but Stormpath should pick one and stick with it (2nd person singular).

**Comment by nbarbettini** [comment]
Didn't even notice I was doing that. Fixed :+1:

**Pull request merged**